File: solutions/beecrowd/1805/1805.lua
while true do
    local line = io.read()

    if line == nil then break end

    for first_s, last_s in string.gmatch(line, "(%d+) (%d+)") do
        first = tonumber(first_s)
        last = tonumber(last_s)

        print(math.tointeger((last - first + 1) * (first + last) / 2))
    end
end

File: solutions/beecrowd/1805/1805.exs
IO.stream(:stdio, :line) |>
Enum.each(fn line ->
    [first, last] = line |>
    String.split()|>
    Enum.map(&String.to_integer/1)

    result = div((last - first + 1) * (first + last), 2)
    IO.puts(result)
end)
